packages/app/navigation/root-stack-navigator.tsx

```diff
@@ -102,8 +106,21 @@ export function RootStackNavigator() {
         />
         <Stack.Screen name="drop" component={DropScreen} />
         <Stack.Screen name="claim" component={ClaimScreen} />
+
         <Stack.Screen name="claimers" component={ClaimersScreen} />
       </Stack.Group>
+      <Stack.Group
```

Comment:
I have added a new Modal Group, see "How 2" for reasons. cc @axeldelafosse  @intergalacticspacehighway